app/javascript/components/DecisionsPage.jsx

```diff
@@ -0,0 +1,154 @@
+import React, {useState, useEffect} from 'react';
+import Form from "react-bootstrap/Form";
+import axios from 'axios';
+import Loader from "./Loader";
+import {createCsrfToken, isEmptyStr, isPresent} from "./helpers/helper";
+import {Button} from "react-bootstrap";
+import Decisions from "./Decisions";
+import {Typeahead} from "react-bootstrap-typeahead";
+
+export const BtnPrimary = ({ text, hidden, onClick, disabled }) =>
+  <Button onClick={onClick} style={{backgroundColor: '#6A1B9A', border: 'none'}} hidden={hidden} disabled={disabled}>
+    {text}
+  </Button>
+
+const onCreateDecision = (decision, setDecisions, setLoaded, setDecision, setError,
+                          setSelectedTemplate, setSelectedUser, setSelectedCollaborators) => {
+  const url = '/create_decision';
+  createCsrfToken()
+  setLoaded(false)
+
+  axios.post(url, { decision })
```

Comment:
@SerhiiBorozenets It might make sense to move API requests to some API_helperrs or some another separate module